compiler/rustc_mir/src/transform/early_otherwise_branch.rs

```diff
@@ -41,8 +41,8 @@ impl<'tcx> MirPass<'tcx> for EarlyOtherwiseBranch {
             .flat_map(|(bb_idx, bb)| {
                 let switch = bb.terminator();
                 let helper = Helper { body, tcx };
```

Comment:
Is it necessary to create a different Helper object here in each iteration?

Reply:
It's not. Fixed.